qemu/tests/cfg/netperf_udp_perf.cfg

```diff
@@ -0,0 +1,59 @@
+- netperf_udp_perf: install setup image_copy unattended_install.cdrom
```

Comment:
Seems this config file shares a lot of same params with netperf.cfg, this case I would recommend to move them into one config file and have udp test as a different variant.

Reply:
@quanwenli What do think about this one?

Reply:
yes, there are some same params with netperf case, but since they are differnece cases, difference test method, I prepare to keep it in seperate one.